platform/server/elasticsearch/Cluster.ts

```diff
@@ -13,7 +13,7 @@ export class Cluster {
     private readonly config: ElasticsearchConfig,
     logger: LoggerFactory
   ) {
-    this.log = logger.get('elasticsearch', 'client', config.clusterType);
+    this.log = logger.get(`elasticsearch::client::${config.clusterType}`);
```

Comment:
Yeah, I don't like this at all, it looks _really_ bad. I'd love to get away from `::` in any external api at least.

In general I'm not convinced this shouldn't be an array internally. It just "feels" like a better api for this. Do we definitely have to `join` then `split` if we make it consistently an array internally? Feels like it could be an array all the way until it becomes formatted in the Layout.